### ISSUE TITLE
Make vampire Rejuvenate remove chloral/sleeptoxin/etc

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#8609]
(WEB REPORT BY: zealotron REMOTE: 158.69.55.2:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 25e4b508298181d88282a40acf22e88f603125b5 
> 
> General description of the issue
> i was chloral'd and it didn't remove it or remove the like drunk random moving thing
> 
> What you expected to happen
> 
> What actually happened
> 
> Steps to reproduce if possible

**Comment by gbasood** [comment]
THANKS SAWBONES FOR DESCRIBING SO LITTLE

**Comment by TheFriggenSawbones** [comment]
no problem

**Comment by TheFriggenSawbones** [comment]
I got hit with an autoinjector full of chloral, and fell asleep somewhere. I tried to mash Rejuvenate assuming it'd cleanse the chloral from my body (I think that's what it's supposed to do right?) and it didn't work - I was still asleep. Even after I got up I still had that like drunk random movement thing going on. I hit rejuvenate a few more times and nothing changed I was still kinda fucked.

So either I don't understand what that vampire ability is supposed to do or it's bugged and doesn't remove chloral from your body
